Solutions/Tomcat/Hunting Queries/TomcatTopURLsClientErrors.yaml

```diff
@@ -0,0 +1,28 @@
+id: 60725e58-4881-11ec-81d3-0242ac130003
+name: Tomcat - Top URLs client errors
+description: |
+  'Query shows URLs list with client errors.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+tactics:
+  - Impact
+  - InitialAccess
+relevantTechniques:
+  - T1498
+  - T1190
+  - T1133
+query: |
+  TomcatEvent
+  | where TimeGenerated > ago(24h)
+  | where toint(HttpStatusCode) >= 400 and toint(HttpStatusCode) <= 499

```

Comment:
same on toint()

Reply:
Query works in Sentinel w/o casting toint().
Casting toint() was added to pass checks. If there is no casting toint() in files then checks fail with error